roles/logging-config/tasks/main.yml

```diff
@@ -1,5 +1,6 @@
 - include: logstash.yml
-  when: logging.forward_type != 'filebeat'
+  when: logging.forward_type|default('filebeat') != 'filebeat'
```

Comment:
should we just kill LSF from here ?  provide a couple of tasks to uninstall it if its there instead ?